### ISSUE TITLE
Mac client menubar app does not turn Winscribe focus to topmost app for Terminal

### ISSUE THREAD

**Issue by d7an** [Windscribe/Desktop-App#107]
Winscribe Desktop for Mac 2.7.10 (Guinea Pig), macOS 13.5.2

To reproduce
1) Start Winscribe Desktop, hide/"close" it
2) Start Terminal.app, making it the active app
3) Click on Winscribe Desktop menubar icon, you'll notice Winscribe hides behind Terminal

For other apps like browser, Winscribe Desktop does shows up above, making itself visible, but not for Terminal.

**Comment by Ivantheallknowing** [comment]
I'm unable to reproduce this behavior.

**Comment by bernerdad** [comment]
Hi.  Can you let us know if this still occurs in our latest guinea pig build?  Thanks!
https://deploy.totallyacdn.com/desktop-apps/2.8.2/Windscribe_2.8.2_guinea_pig.dmg

**Comment by d7an** [comment]
> Hi. Can you let us know if this still occurs in our latest guinea pig build? Thanks! https://deploy.totallyacdn.com/desktop-apps/2.8.2/Windscribe_2.8.2_guinea_pig.dmg

I installed this 2.8.2, it didn't solve the issue of not raising above Terminal.app

I've found Terminal seems to be "special", I came across some other menubar app that couldn't raise their app to the above Terminal.  Having said that, there're definitely apps that could raise above Terminal.app, to name a few.

**Comment by d7an** [comment]
> Hi. Can you let us know if this still occurs in our latest guinea pig build? Thanks! https://deploy.totallyacdn.com/desktop-apps/2.8.2/Windscribe_2.8.2_guinea_pig.dmg

I found the caused!!  When Terminal/iTerm2 has the "Secure Keyboard Entry" option from Terminal/iTerm2 menu checked, one will experience what I described at the beginning.  Unchecking this option "fixes" the "issue".

I'm not sure how Windscribe app is special in the way it opens the menubar app menu, other apps I've like AdGuard, MenuBar Stats, Little Snitch (to name a few) doesn't have such problem, they'll open their menubar menu happily even when the Secure Keyboard Entry is checked.

Anyway, at least I'm not puzzled by this weirness.
